src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -50,7 +63,7 @@ import {
   ]
 })
 export class SkyTabsetComponent
-  implements AfterContentInit, AfterViewInit, OnDestroy, OnChanges {
+  implements OnInit, AfterContentInit, AfterViewInit, OnDestroy, OnChanges {
```

Comment:
Can we move this `OnInit` to be in the alphabetical order?

Reply:
This is something we can discuss as a group, but I tend to put implementations in the order that they are fired (similar to how we order the same methods in the class).

Reply:
Ok. Sounds good.